### ISSUE TITLE
Speedtest broken

### ISSUE THREAD

**Issue by Sandvich** [PhasecoreX/PCXCogs#45]
When running the speed test, I get the following error (in the docker logs)
```
[2021-04-12 16:25:38] [ERROR] red: Exception in command 'netspeed'
Traceback (most recent call last):
  File "/data/venv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/data/cogs/CogManager/cogs/netspeed/netspeed.py", line 31, in netspeed
    s = speedtest.Speedtest(secure=True)
  File "/data/cogs/Downloader/lib/speedtest.py", line 1091, in __init__
    self._shutdown_event = shutdown_event
  File "/data/cogs/Downloader/lib/speedtest.py", line 1173, in get_config
    upload = get_attributes_by_tag_name(root, 'upload')
ValueError: invalid literal for int() with base 10: ''
```
I'm not sure that this is a cog issue though, from the traceback it looks like a library issue?

**Comment by PhasecoreX** [comment]
Yes, this was fixed a couple of days ago with #41, make sure the cog is up to date, then also restart the bot.

**Comment by Sandvich** [comment]
Ah, it was the restarting that I missed. My mistake! Thanks :)
